tests/testthat/helper.R

```diff
@@ -80,3 +80,12 @@ local_proj_set <- function(envir = parent.frame()) {
 cat_file <- function(...) {
   cat(brio::read_file(file.path(...)))
 }
+
+#' Helper function for skip testing.
+#' @param args Character vector, arguments to the `cargo` command. Pass to [processx::run()]'s args param.
+skip_if_cargo_bin <- function(args = "--help") {
+  if (processx::run("cargo", args, error_on_status = FALSE)$status != 0) {
+    message <- paste0("`cargo ", paste0(args, collapse = " "), "` is not available.")
```

Comment:
This would be awesome to have in `write_license_note()`

Reply:
> This would be awesome to have in `write_license_note()`

I thought of that too, but no such checking mechanism seems to exist in other functions that call `cargo`.

Reply:
I suppose that goes back to `rust_sitrep()` https://github.com/extendr/rextendr/issues/168

Reply:
I added a helper function to check cargo sub command.

I don't know if this will be useful for #168.